base_tier_validation/models/tier_review.py

```diff
@@ -20,6 +20,14 @@ class TierReview(models.Model):
     )
     model = fields.Char(string="Related Document Model", index=True)
     res_id = fields.Integer(string="Related Document ID", index=True)
+    record_id = fields.Reference(
```

Comment:
@LoisRForgeFlow I would like to have your review on this. Thanks

I kept former fields for compatibility.